Readme.md

```diff
@@ -178,6 +184,11 @@ Wait until the element `selector` is present e.g. `.wait('#pay-button')`
 #### .wait(fn[, arg1, arg2,...])
 Wait until the `fn` evaluated on the page with `arg1, arg2,...` returns `true`. All the `args` are optional. See `.evaluate()` for usage.
 
+#### .wait('downloads-complete')
+Wait until all downloads are in a state of `'completed'`, `'interrupted'`, or `'cancelled'`
```

Comment:
i don't think it's a good idea to overload `nightmare.wait(str)` in this way. It sounds great semantically, but if we have custom tags like `<download-complete></download-complete>`, we'll run into problems.